sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/test/Microsoft.Azure.Webjobs.Extensions.SignalRService.E2ETests/Microsoft.Azure.Webjobs.Extensions.SignalRService.E2ETests.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Take a look at https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template/Azure.Template/tests to see how we structure/name our test projects/directories.

If it isn't too much of a change, it might be worth following the same pattern.